producer.properties

```diff
@@ -0,0 +1,8 @@
+bootstrap.servers localhost:9092
```

Comment:
Same as before. Also, Could I have made them Into a single config file? Does it make sense to have a different config for different features ? For a complex application, I could have hundreds of config files then, right ?

Reply:
Merged both into single client.properties file in a newer commit